Loop/Views/Presets/Components/RepeatOptionsView.swift

```diff
@@ -28,9 +30,9 @@ struct RepeatOptionView: View {
         } else {
             HStack(spacing: 4) {
                 ForEach(PresetScheduleRepeatOptions.allCases, id: \.rawValue) { day in
-                    Text(String(day.description.first!))
-                        .font(.system(size: 12))
-                        .frame(width: 20, height: 20)
+                    Text(String(day.veryShortDescription))
+                        .font(.system(size: dayTextSize))
+                        .frame(width: dayTextSize+8, height: dayTextSize+8)
```

Comment:
Would a padding modifier not work here?

Reply:
I think with kerning and such, the circles would vary in size.

Reply:
in that case you can use a monospaced modifier.
```suggestion
                        .monospaced()
                        .padding(8)
```